bulk-model-sync-gradle-test/.gitignore

```diff
@@ -0,0 +1,49 @@
+.gradle
+build/
+!gradle/wrapper/gradle-wrapper.jar
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### IntelliJ IDEA ###
+.idea/modules.xml
+.idea/jarRepositories.xml
+.idea/compiler.xml
+.idea/libraries/
+*.iws
+*.iml
+*.ipr
+out/
+!**/src/main/**/out/
+!**/src/test/**/out/
+
+### Eclipse ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+bin/
+!**/src/main/**/bin/
+!**/src/test/**/bin/
+
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+
+### VS Code ###
+.vscode/
+
+### Mac OS ###
+.DS_Store
+/test-repo/languages/GraphLang/classes_gen/
+/test-repo/languages/GraphLang/generator/
+/test-repo/languages/GraphLang/source_gen/
+/test-repo/languages/GraphLang/source_gen.caches/
+/test-repo/solutions/GraphSolution/source_gen/
+/test-repo/solutions/GraphSolution/classes_gen/
+/test-repo/solutions/GraphSolution/source_gen.caches/
```

Comment:
This could be reduced with some wildcards. Moreover, that would then also be resilient against language renamings.